data/json/recipes/recipe_others.json

```diff
@@ -932,6 +932,23 @@
   ]
 },{
   "type" : "recipe",
+  "result": "crude_firestarter",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_TOOLS",
+  "skill_used": "electronics",
+  "difficulty": 0,
```

Comment:
Please bump the difficulty to 1. It's got to be reliable and reusable, after all.